apps/web/src/components/(marketing)/footer.tsx

```diff
@@ -10,6 +10,19 @@ import { cn } from '@documenso/ui/lib/utils';
 export type FooterProps = HTMLAttributes<HTMLDivElement>;
 
 export const Footer = ({ className, ...props }: FooterProps) => {
+  const socialLinks = [
```

Comment:
Move these above the component and make them shouty snake case so we aren't instantiating them on every render (not that it costs much).

So `socialLinks` => `SOCIAL_LINKS`

Reply:
Done 🚀